### PR TITLE
Allow containers/common to use internal TF

### PR DESCRIPTION
**Pull request by lsm5**
TODO:

- [ ] Write new tests or update the old ones to cover new functionality.
- [ ] Update doc-strings where appropriate.
- [ ] Update or write new documentation in `packit/packit.dev`.
- [ ] ‹fill in›

Fixes

Related to

Merge before/after

RELEASE NOTES BEGIN

Packit now supports automatic ordering of ☕ after all checks pass.

RELEASE NOTES END

### CODE REVIEW

File: secrets/packit/prod/packit-service.yaml.j2 [hunk 1]
```diff
@@ -26,6 +26,7 @@ enabled_projects_for_internal_tf:
   - github.com/RedHat-SP-Security/keylime-selinux
   - github.com/containers/aardvark-dns
   - github.com/containers/buildah
+  - github.com/containers/common
```
Comment (nforro) [inline secrets/packit/prod/packit-service.yaml.j2:29]: Could you add it to `stg` as well, so the lists don't diverge?

**Review by nforro — APPROVED** [review]
Thanks!

File: secrets/packit/prod/packit-service.yaml.j2 (hunk 1 above)
Comment (lsm5) [reply secrets/packit/prod/packit-service.yaml.j2:29]: done. I thought stg was only for projects that wanted to use stage so I ignored it earlier. Thanks!

**Comment by nforro** [comment]
Service has to be redeployed after the PR is merged, I'm going to do it ASAP.

**Pull request merged**